src/image-copy/azext_imagecopy/custom.py

```diff
@@ -21,9 +21,17 @@ def imagecopy(cmd, source_resource_group_name, source_object_name, target_locati
               target_resource_group_name, temporary_resource_group_name='image-copy-rg',
               source_type='image', cleanup='false', parallel_degree=-1, tags=None, target_name=None,
               target_subscription=None, export_as_snapshot='false', timeout=3600):
+    # If --cleanup is set, forbid using an existing temporary resource group name.
+    # It is dangerous to clean up an existing resource group.
+    cli_cmd = prepare_cli_command(['group', 'exists', '-n', temporary_resource_group_name],
```

Comment:
Where is the code for checking `If --cleanup is set`?

Reply:
Fixed